gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageGrpcReadChannel.java

```diff
@@ -759,6 +758,19 @@ private void cancelCurrentRequest() {
     contentChannelEndOffset = -1;
   }
 
+  private void drainIterator() {
+    if (resIterator == null) {
+      return;
+    }
+    try {
+      while (resIterator.hasNext()) {
+        resIterator.next();
+      }
+    } catch (Exception e) {
+      logger.atWarning().withCause(e).log("Exception while draining the iteration on cancellation");
```

Comment:
Suggest we change this to Info as it is expected to raise exception

Reply:
Done. `expected exception` is  counter-intuitive. We should explore a clear solution, possibly embed this logic under `cancel` grpc api